### ISSUE TITLE
lost posts when working offline

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#338]
```
What's the problem?
I save the progress of a post in offline mode then come back and can not locate 
the post




What browser are you using?
Chrome




What version of ScribeFire are you running?
1.5 (latest download) 
```

-----
Original issue reported on code.google.com by `unvl...@gmail.com` on 15 May 2011 at 9:47

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 3 Nov 2011 at 3:20
* Changed state: **Accepted**
* Added labels: **Milestone-4.1**

**Comment by GoogleCodeExporter** [comment]
```
I use Firefox, and cannot retrieve posts started offline. There is no button to 
go to any local database. Can I find it using Terminal somehow?
```

-----
Original comment by `kiai...@gmail.com` on 13 Jan 2012 at 4:14

**Comment by GoogleCodeExporter** [comment]
```
Note: The titles of new posts do not show up in the list of posts in the left 
column.
```

-----
Original comment by `kiai...@gmail.com` on 13 Jan 2012 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
what blog type are you using and are you using scribefire-next?
```

-----
Original comment by `scribefi...@gmail.com` on 8 Apr 2014 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
Oh dear, this was over 2 years ago. I think I was using Blogger at the time. 
Now I'm using WordPress.
```

-----
Original comment by `kiai....@gmail.com` on 9 Apr 2014 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:08
* Changed state: **Fixed**
